e2e/cypress/integration/custom_status/custom_status_spec.js

```diff
@@ -0,0 +1,557 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @custom_status
+
+import {openCustomStatusModal} from './helper';
+
+describe('Custom Status modal', () => {
+    let currentUser;
+    before(() => {
+        cy.apiUpdateConfig({TeamSettings: {EnableCustomUserStatuses: true}});
+
+        // # Login as test user and visit channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, user, channel}) => {
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+            currentUser = user;
+        });
+    });
+
+    describe('MM-T3851 Custom status CTAs for new users', () => {
```

Comment:
Please move to its own file `custom_status_1_spec.js`

```suggestion
    describe('Custom Status - CTAs for New Users', () => {
```